CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.2.1
+ - `context.showLoaderOverlay` and `context.hideLoaderOverlay` marked as depricated. You should use `context.loaderOverlay.show` and `context.loaderOverlay.hide`
+ - widget that you pass to `context.loaderOverlay.show` will be shown instead of `widgetOverlay`
```

Comment:
> - widget that you pass to `context.loaderOverlay.show` will be shown instead of `widgetOverlay`

Change to `- widget that you pass (if so) to `context.loaderOverlay.show` will be shown instead of `widgetOverlay`